### ISSUE TITLE
[Grav/Helium] Invalid HTML : main element as a descendant of the section element

### ISSUE THREAD

**Issue by Fazarel** [gantry/gantry5#2834]
**Grav** v1.7.17 - Admin v1.10.17
**Gantry 5** Framework v5.4.37
Theme: **Helium** (v5.4.37 / g5_helium)

Hello,

I get an error in [validator.w3.org](https://validator.w3.org): "**The `main` element must not appear as a descendant of the `section` element.**"

The message refers to this part of code:
```
<section id="g-container-main" class="g-wrapper">
<div class="g-container"> <div class="g-grid"> <div class="g-block size-100">
<main id="g-mainbar">
<div class="g-grid"> <div class="g-block size-100"> <div class="g-content">
<p><img title="..." src="...jpg" /></p>
</div> </div> </div>
</main>
</div> </div> </div>
</section>
```

The page has a very basic layout, with Navigation, Container and Footer sections.
![Gantry_Templates](https://user-images.githubusercontent.com/35377747/125170146-be1b8b00-e1ad-11eb-9b7f-d1ff8f3173c9.jpg)

The content of the page consists of a simple image.

Can I change anything to get a valid HTML code?

Thanks in advance for your help and by the way, thanks for your work.

**Comment by mahagr** [comment]
Good catch. You can probably just change the presets not to use main in here:

https://github.com/gantry/gantry5/blob/develop/themes/helium/common/layouts/default.yaml#L87-L89
https://github.com/gantry/gantry5/blob/develop/themes/helium/common/layouts/home_-_particles.yaml#L74-L76

Then just reload the presets from layout manager and remember to check the option to keep the particles.

**Comment by mahagr** [comment]
@hexplor This is something you should look at, especially with the new themes (not sure how many sites the fix would break in existing themes)...
